blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/serializers/NumericSerializer.java

```diff
@@ -55,6 +48,7 @@
     public static AbstractSerializer<TimerRollup> timerRollupInstance = new TimerRollupSerializer();
     public static AbstractSerializer<SetRollup> setRollupInstance = new SetRollupSerializer();
     public static AbstractSerializer<GaugeRollup> gaugeRollupInstance = new GaugeRollupSerializer();
+    public static AbstractSerializer<EnumRollup> enRollupInstance = new EnumRollupSerializer();
```

Comment:
NIT:  why not just spell out "enumRollupInstance"?  it's only 2 letters more and it reads clearer.